metpy/gridding/tests/test_points.py

```diff
@@ -0,0 +1,160 @@
+# Copyright (c) 2008-2015 MetPy Developers.
+# Distributed under the terms of the BSD 3-Clause License.
+# SPDX-License-Identifier: BSD-3-Clause
+
+from __future__ import division
+
+from metpy.gridding.points import (get_points_within_r, get_point_count_within_r,
+                                   get_boundary_coords, get_xy_steps, get_xy_range,
+                                   generate_grid, generate_grid_coords)
+
+from numpy.testing import assert_array_almost_equal
+
+import numpy as np
+
+
+def test_get_points_within_r():
+    r"""Tests get points within a radius function"""
+
+    x = list(range(10))
+    y = list(range(10))
+
+    center = [1, 5]
+
+    radius = 5
+
+    matches = get_points_within_r(center, list(zip(x, y)), radius).T
+
+    truth = [[1, 1], [2, 2], [3, 3], [4, 4], [5, 5]]
+
+    assert_array_almost_equal(truth, matches)
+
+
+def test_get_point_count_within_r():
+    r"""Tests get point count within a radius function"""
+
+    x = list(range(10))
+    y = list(range(10))
+
+    center1 = [1, 5]
+    center2 = [12, 10]
+
+    radius = 5
+
+    count = get_point_count_within_r([center1, center2], list(zip(x, y)), radius)
+
+    truth = np.array([5, 2])
+
+    assert_array_almost_equal(truth, count)
+
+
+def test_get_boundary_coords():
+    r"""Tests get spatial corners of data positions function"""
+
+    x = list(range(10))
+    y = list(range(10))
+
+    bbox = get_boundary_coords(x, y)
+
+    east_truth = 9
+    north_truth = 9
+    south_truth = 0
+    west_truth = 0
+
+    assert_array_almost_equal(north_truth, bbox['north'])
```

Comment:
These aren't actually arrays, so why use almost equal? In fact, they're integers, so why bother with the amost part? You could just put the coords in a dictionary and let pytest handle it for you:
```python
truth = dict(east=9, north=9, south=0, west=0)
assert bbox == truth
```